src/hypergraph.jl

```diff
@@ -309,11 +309,11 @@ function remove_hyperedge!(h::Hypergraph, e::Int)
 end
 
 """
-	clean!(h::Hypergraph)
-Removes all verticies with degree 0 and all hyperedges with size 0.
+    prune_hypergraph!(h::Hypergraph)
+Removes all vertices with degree 0 and all hyperedges of size 0.
 """
 
-function clean!(h)
```

Comment:
The state-of-the art in Julia is to deprecate a function in one release and remove it in another.
Hence it should be:
```julia
clean!(h::Hypergraph) = prune_hypergraph!(h)
@deprecate clean!(h::Hypergraph) prune_hypergraph!(h::Hypergraph)
```